src/coreL2/CLIL2Interface/CLIL2Interface.h

```diff
@@ -0,0 +1,101 @@
+/* ***************************************/
+/* Copyright Notice                      */
+/* Copyright(c)2020 5G Range Consortium  */
+/* All rights Reserved                   */
+/*****************************************/
+
+#ifndef INCLUDED_MAC_CONFIG_REQUEST_H
+#define INCLUDED_MAC_CONFIG_REQUEST_H
+
+#include <iostream>
+#include <fstream>		//File stream
+#include <string.h>
+#include <vector>
+#include <cstdlib>
+#include <mutex>
+using namespace std;
+
+#include "../../common/lib5grange/lib5grange.h"
+#include "../../common/libMac5gRange/libMac5gRange.h"
+#include "../SystemParameters/DynamicParameters.h"
+using namespace lib5grange;
+
+/**
+ * @brief Class to interact with CLI
+ */
+class CLIL2Interface{
+private:
+	bool macStartCommandFlag;			//Flag to control CLI's request of MacStartCommand
```

Comment:
Apenas sugestão de nome..... Fique a vontade se não quiser alterar... acho que "signal" soa melhor que "flag"
.......
.......
bool macStopCommandSignal;			//Signal to control CLI's request of MacStopCommand
bool macConfigRequestCommandSignal;